lib/ansible/plugins/inventory/vmware_vm_inventory.py

```diff
@@ -27,6 +27,9 @@
       - "vSphere Automation SDK - For tag feature"
       - "vCloud Suite SDK - For tag feature"
     options:
+        datacenter:
```

Comment:
Also add an example of this [https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/vmware/vmware_inventory.rst].

Reply:
Yes. Add an example using new options to that page.

Reply:
Done. [Check it out](https://github.com/ansible/ansible/pull/52782/files#diff-245a3a9549a5cf501e2a0cf86793e9c6) @Akasurde